### ISSUE TITLE
Color contrast for open graph listing images fails accessibility

### ISSUE THREAD

**Issue by whitep4nth3r** [forem/forem#16163]
**Describe the bug**

Open Graph images for listing pages do not pass AAA accessibility standards for colour contrast.

**To Reproduce**

View this open graph image: https://hcti.io/v1/image/4ba2409f-ed0c-448b-bbe0-047a009a1814
(Taken from this URL https://dev.to/listings/events/dev-twitch-stream-with-dan-abramov-maggie-appleton-34c6)

Observe colour contrast scores:

![A screenshot showing colour contrast scores for white and the chosen pink font](https://user-images.githubusercontent.com/52798353/149940595-e5f77274-6e23-41ad-8692-9e914f81f634.png)

**Expected behavior**

New font colour will contrast with a higher ratio (greater than 4.5) against the #ffffff background.

**Additional context**

Extra scoring from https://contrastchecker.com/
![Scoring from contrastchecker.com](https://user-images.githubusercontent.com/52798353/149941197-a32ae625-4987-44ed-93af-238ca33fd28a.png)

Some reading around contrast ratio from https://webaim.org/articles/contrast/#ratio.

**Comment by nickytonline** [comment]
Thanks for opening the issue @whitep4nth3r! That definitely fails colour contrast. 😬

**Comment by nickytonline** [comment]
This requires investigation on the backend. I'm not sure what we're currently using to generate the social image cards.
